gateway/modules/filestore/store.go

```diff
@@ -65,6 +69,14 @@ func (m *Module) SetConfig(conf *config.FileStore) error {
 		}
 	}
 
+	// set fileStore secret for aws and gcp
+	connection := strings.Split(conf.Conn, ".")
```

Comment:
1) You are using the wrong variable for splitting, you should use **conf.Secret**
2) Make a function that does the splitting, it will return secretName,secretKey,isSecretExists
3) As the function is same for both crud & filestore. Write the above function in utils folder